File: DPBO-praktikum/2/Java/CivitasAkademik.java
public class CivitasAkademik extends Human {
    protected String universitas;
    protected String email;

    // Konstruktor
    public CivitasAkademik(String NIK, String nama, String gender, String universitas, String email) {
        super(NIK, nama, gender);
        this.universitas = universitas;
        this.email = email;
    }

    // Getter untuk universitas
    public String getUniversitas() {
        return universitas;
    }

    // Getter untuk email
    public String getEmail() {
        return email;
    }
}

File: DPBO-praktikum/2/CPP/CivitasAkademik.cpp
#include "Human.cpp"
using namespace std;

class CivitasAkademik : public Human {
protected:
    string universitas;
    string email;

public:
    // Konstruktor
    CivitasAkademik(string NIK, string nama, string gender, string universitas, string email)
        : Human(NIK, nama, gender) {
        this->universitas = universitas;
        this->email = email;
    }

    // Getter untuk universitas
    string getUniversitas() {
        return universitas;
    }

    // Getter untuk email
    string getEmail() {
        return email;
    }
};